### PR TITLE
resolve #678

### LINKED ISSUE #678: Can't set Dialogflow language
- BotMan Version: 2.0

### Description:

Can't set Dialogflow language - en is hardcoded
### Steps To Reproduce:

### PR DESCRIPTION
**Pull request by luckyraul**

### CODE REVIEW

File: src/Middleware/ApiAi.php [hunk 1]
```diff
@@ -30,21 +30,24 @@ class ApiAi implements MiddlewareInterface
...
      * Create a new Wit middleware instance.
-     * @param string $token wit.ai access token
+     * @param string $token api.ai access token
+     * @param string $lang language
      * @return ApiAi
      */
-    public static function create($token)
+    public static function create($token, $lang = 'en')
     {
         return new static($token, new Curl());
```
Comment (mpociot) [inline src/Middleware/ApiAi.php]: You forgot to pass the `$lang` variable to the constructor.

**Comment by luckyraul** [comment]
@mpociot  check

**Comment by luckyraul** [comment]
@mpociot  accept please

**Comment by luckyraul** [comment]
@mpociot merge please

**Pull request merged**